src/main/kotlin/nl/info/zac/app/zaak/converter/RestZaakConverter.kt

```diff
@@ -147,6 +144,8 @@ class RestZaakConverter @Inject constructor(
                 StatusTypeUtil.isHeropend(statustype) -> EnumSet.of(ZaakIndicatie.HEROPEND)
                 zaak.isOpgeschort -> EnumSet.of(ZaakIndicatie.OPSCHORTING)
                 zaak.isVerlengd -> EnumSet.of(ZaakIndicatie.VERLENGD)
+                shouldSetOntvangstbevestigingNietVerstuurdIndication(zaak, statustype) ->
+                    EnumSet.of(ZaakIndicatie.ONTVANGSTBEVESTIGING_NIET_VERSTUURD)
```

Comment:
```suggestion
                    EnumSet.of(ONTVANGSTBEVESTIGING_NIET_VERSTUURD)
```

Maybe personal but there is no need to add the class name in here because you can just import it. For me this makes the code a bit more readable.